content/packages.md

```diff
@@ -31,11 +32,124 @@ my-package
+Package.onUse(function(api) {
+  // ...
+
+  api.addAssets([
+    'font/OpenSans-Light-webfont.eot',
+    'font/OpenSans-Light-webfont.svg',
+    'font/OpenSans-Light-webfont.ttf',
+    'font/OpenSans-Light-webfont.woff',
+    'font/OpenSans-Regular-webfont.eot',
+    'font/OpenSans-Regular-webfont.svg',
+    'font/OpenSans-Regular-webfont.ttf',
+    'font/OpenSans-Regular-webfont.woff',
+  ], 'client');
+
+  // ...
+});
+```
+
+As you can see, the `addFiles` and `addAssets` functions allow you to pass a list of files as the first argument, and an architecture (or array of architectures) as the second argument. For more information about this, see the section about architectures below.
+
+## Package dependencies
+
+Another very important feature of Meteor packages is the ability to register dependencies on other packages. This is done via `api.use` and `api.imply`.
+
+`api.use` is for recording the dependencies of your package. These dependencies are only available inside the package. However, Meteor's package system is single-loading so no two packages in the same app can have dependencies on conflicting versions of a single package. Read more about that in the section about version constraints below.
+
+```js
+// Example of using api.use to register dependencies internal to the package
+api.use([
+  'ecmascript',
+  'check',
+  'ddp',
+  'underscore',
+  'aldeed:simple-schema@1.3.3',
+  'mdg:validation-error@0.1.0'
+]);
+```
+
+`api.imply`, on the other hand, does not include the package as an internal dependency; instead, it makes these packages available to the user of the package. In the example below, we use `api.imply` to include Flow Router in the `todos-lib` package, so that any user of `todos-lib` also gets Flow Router. This can be helpful to avoid keeping long lists of dependencies up to date by creating meta-packages that encapsulate many dependencies of your app at once.
+
+```js
+// Example of using api.imply to make a meta-package
+// Taken from the todos-lib package in the Todos example app
+api.imply([
+  'kadira:flow-router@2.7.0',
+  'kadira:blaze-layout@2.2.0',
+  'arillo:flow-router-helpers@0.4.5',
+  'zimme:active-route@2.3.0',
+]);
+```
+
+`api.use` and `api.imply` can also take an extra argument to specify the architecture on which these dependencies should be used - this way, you can register a dependency on only the server version of HTTP, for example. This can be helpful when you are trying to reduce the side of your client-side app bundle.
+
+### You can't depend on a specific Meteor version
+
+Note that the Meteor release version number is mostly a marketing artifact, so it doesn't follow SemVer. This means packages can only depend on specific versions of the packages inside a Meteor release, but can't depend on a specific release itself. We have a helpful shorthand api called `versionsFrom` that handles this for you by automatically filling in package version numbers from a particular release:
```

Comment:
- "marketing artifact"? -- maybe "useful convenience that doesn't immediately tell you anything about the version numbers of the various packages that make up Meteor core (and which follow SemVer)"
- Link to what SemVer means?
- "We have" -> I think "there is" is a better tone?